packages/shared/src/settings/borderRadiusExtended.ts

```diff
@@ -58,28 +58,52 @@ export const getExtendedBorderRadiusSettings = (options?: BorderRadiusSettingsTy
                         type: 'input',
                         label: 'Top Left',
                         rules: [numericalOrPixelRule],
-                        onChange: (bundle: Bundle): void => appendUnit(bundle, topLeftId),
+                        onChange: (bundle) => {
+                            if (bundle.getBlock(topLeftId)?.value === '') {
+                                bundle.setBlockValue(topLeftId, '0px');
+                            } else {
+                                appendUnit(bundle, topLeftId);
+                            }
+                        },
                     },
                     {
                         id: topRightId,
                         type: 'input',
                         label: 'Top Right',
                         rules: [numericalOrPixelRule],
```

Comment:
Actually not "approved" because it collides with the `numericalOrPixelRule` rule

Reply:
I think that this should still be correct.
I clear the input -> it gets set to 0px -> I clear again -> field is empty but incorrect. Do we want to allow the user to save an empty string?